tonic/models/vic/vic.py

```diff
@@ -67,8 +67,18 @@ def run(self, global_param, logdir=None):
 
         return self.returncode
 
-    def _call_vic(self, *args):
-        vic_args = [self.executable] + [a for a in args]
+    def _call_vic(self, *args, mproc_num=None):
+
+	vic_args = []
```

Comment:
Yep, sorry about that. I made these changes several weeks ago before I changed my default spacing.